.eslintrc.json

```diff
@@ -34,7 +34,8 @@
     "default-case": 0,
     "import/no-extraneous-dependencies": 0,
     "no-cond-assign": 0,
-    "array-callback-return": 0,
-    "function-paren-newline": 0
+    "import/no-unresolved": 0,
+    "no-await-in-loop": 0,
```

Comment:
we have loops where await is used following `if` conditions. There is no clean way to rework it in `Promise.all`